### PR TITLE
Fix builds with CUDA disabled

### LINKED ISSUE #60: build failure on LC toss3 machines
When using the umpire host-config to build chai, i get an error.

The command:
```
cmake -C ../src/tpl/umpire/host-configs/toss_3_x86_64_ib/clang_4_0_0.cmake ..
```
gives:
```
CMake Error at blt/cmake/thirdparty/SetupCUDA.cmake:6 (enable_language):
  The CMAKE_CUDA_COMPILER:

    /usr/tce/packages/cuda/cuda-9.1.85/bin/nvcc

  is not a full path to an existing compiler tool.

  Tell CMake where to find the compiler by setting either the environment
  variable "CUDACXX" or the CMake cache entry CMAKE_CUDA_COMPILER to the full
  path to the compiler, or to the compiler name if it is in the PATH.
Call Stack (most recent call first):
  blt/cmake/thirdparty/SetupThirdParty.cmake:77 (include)
  blt/SetupBLT.cmake:100 (include)
  CMakeLists.txt:102 (include)


-- Configuring incomplete, errors occurred!
See also "/g/g15/settgast/workspace/Codes/geosx/CHAI/build-quartz/CMakeFiles/CMakeOutput.log".
See also "/g/g15/settgast/workspace/Codes/geosx/CHAI/build-quartz/CMakeFiles/CMakeError.log".
```
If disable cuda, I get the following error:
```
CMake Error: Error required internal CMake variable not set, cmake may not be built correctly.
Missing variable is:
CMAKE_CUDA_COMPILE_WHOLE_COMPILATION
CMake Error: Error required internal CMake variable not set, cmake may not be built correctly.
Missing variable is:
CMAKE_CUDA_DEVICE_LINK_EXECUTABLE
-- Generating done
-- Build files have been written to: /g/g15/settgast/workspace/Codes/geosx/CHAI/build-quartz
```
I get this last error on many platforms using various hostconfigs. The August20 snapshot doesn't have these problems.

### PR DESCRIPTION
**Pull request by davidbeckingsale**
Fixes #60

### CODE REVIEW

**Comment by davidbeckingsale** [comment]
@rrsettgast give this a whirl!

File: examples/CMakeLists.txt [hunk 1]
```diff
@@ -41,10 +41,19 @@
 # POSSIBILITY OF SUCH DAMAGE.
 #######################################################################
 
+set (chai_umpire_example_depends
+  chai)
+
+if (ENABLE_CUDA)
+  set (chai_umpire_example_depends
+    ${chai_umpire_example_depends{
```
Comment (rrsettgast) [inline examples/CMakeLists.txt]: shouldn't this be:
` ${chai_umpire_example_depends}`?

Comment (davidbeckingsale) [inline examples/CMakeLists.txt]:
```suggestion
    ${chai_umpire_example_depends}
```

Comment (davidbeckingsale) [reply examples/CMakeLists.txt]: Should be fixed!

**Pull request merged**